source/adios2/CMakeLists.txt

```diff
@@ -66,14 +67,22 @@ if(ADIOS2_HAVE_SysVShMem)
   target_sources(adios2 PRIVATE toolkit/capsule/shmem/ShmSystemV.cpp)
 endif()
 
+if(ADIOS2_HAVE_ZeroMQ)
+  target_sources(adios2 PRIVATE
+    toolkit/transport/wan/WANZmq.cpp
+  )
+  target_link_libraries(adios2 PRIVATE ZeroMQ::ZMQ)
+endif()
+
 if(ADIOS2_HAVE_DataMan)
   target_sources(adios2 PRIVATE
     engine/dataman/DataManReader.cpp
     engine/dataman/DataManWriter.cpp
   )
-  target_link_libraries(adios2 PRIVATE dataman)
+  target_link_libraries(adios2 PRIVATE NLohmannJson)
```

Comment:
The build error is from the JSON library only being included if DataMan was enabled, which is off by default on Windows.